components/shared_code/src/shared_data_model/subjects.py

```diff
@@ -7,35 +7,99 @@
         name="Development environment",
         description="A software development and/or maintenance environment, consisting of infrastructure, pipelines, "
         "and tools needed to build, test, and deploy software.",
+        subjects={
+            "development_environment_performance": Subject(
+                name="Performance test environment",
+                description="A performance test environment, consisting of infrastrcture, pipelines, and tools "
+                "needed to test the performance of software.",
+                metrics=[
+                    "performancetest_duration",
+                    "performancetest_stability",
+                ],
+            ),
+            "development_environment_pipelines": Subject(
+                name="Pipelines",
+                description="A set of pipelines to build, test, and deploy software.",
+                metrics=[
+                    "failed_jobs",
+                    "pipeline_duration",
+                    "unused_jobs",
+                ],
+            ),
+            "development_environment_tool": Subject(
+                name="Development tool",
+                description="A software development and/or maintenance tool.",
+                metrics=[
+                    "source_version",
+                    "software_version",
+                ],
+            ),
+        },
         metrics=[
-            "failed_jobs",
-            "job_runs_within_time_period",
-            "merge_requests",
-            "performancetest_duration",
-            "pipeline_duration",
-            "software_version",
             "source_up_to_dateness",
-            "source_version",
-            "unmerged_branches",
-            "unused_jobs",
         ],
     ),
     "process": Subject(
         name="Process",
-        description="A software development and/or maintenance process.",
+        description="A software development, maintenance, and/or operations process.",
+        subjects={
+            "process_backlog": Subject(
+                name="Backlog management",
+                description="A process to manage the product backlog.",
+                metrics=["user_story_points", "velocity"],
```

Comment:
```suggestion
                metrics=[
                    "user_story_points",
                    "velocity",
                ],
```

Reply:
Apparently, `ruff format` doesn't care for being consistent across lists :-(